test/plain-list/AutosuggestApp.js

```diff
@@ -48,6 +48,13 @@ export const shouldRenderSuggestions = sinon.spy(value => {
   return value.trim().length > 0 && value[0] !== ' ';
 });
 
+export const toggleAlwaysRenderSuggestions = sinon.spy((toggle, callback) => {
+  app.setState({
+    alwaysRenderSuggestions: toggle,
```

Comment:
"toggle" confuses me here, as I normally think about toggle as "reverse the current state". How about `setAlwaysRenderSuggestions` instead?